### PR TITLE
Feature/validation subscriber

### PR DESCRIPTION
**Pull request by wilsonianb**
Works with `PEERS_API_URL=http://54.201.174.180:1234`, which is defined in circleci

### CODE REVIEW

File: docker-compose.yml [hunk 1]
```diff
@@ -8,5 +8,6 @@ webapp:
     - postgres
   environment:
     - DATABASE_URL=postgres://postgres:postgres@postgres/postgres
+    - PEERS_API_URL
```
Comment (stevenzeiler) [inline docker-compose.yml]: does PEERS_API_URL here need a default?

Comment (wilsonianb) [inline docker-compose.yml]: I think values are optional.
https://docs.docker.com/compose/yml/
I added it so I could run locally with `PEERS_API_URL=http://54.201.174.180:1234 docker-compose up`

**Comment by wilsonianb** [comment]
:+1:

**Comment by stevenzeiler** [comment]
:+1: Any evidence that the websocket subscriber performs better than the old  log-based `validation-tracker`?

**Comment by wilsonianb** [comment]
I think we'd have a better idea at 5pm today assuming we don't experience outages on production or staging.

**Pull request merged**